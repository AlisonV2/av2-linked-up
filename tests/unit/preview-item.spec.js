import { shallowMount } from '@vue/test-utils';
import PreviewItem from '@/components/admin/PreviewItem';

/**
 * @module PreviewItemTest
 */
describe('PreviewItem.vue', () => {
  it('Checks if removePreview event is called', () => {
    const component = shallowMount(PreviewItem, {
      props: {
        image: 'https://via.placeholder.com/150',
        index: '1',
      },
    });
    component.find('.gallery-card').trigger('click');
    expect(component.emitted()).toHaveProperty('removePreview');
  });
});
