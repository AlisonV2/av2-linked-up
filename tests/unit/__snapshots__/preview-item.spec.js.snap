// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviewItem.vue PreviewItem Snapshot 1`] = `
<div
  class="card gallery-card"
>
  <div
    class="card-overlay"
  />
  <div
    class="card-details fade-in-top text-center"
  >
    <i
      class="bi bi-trash img-icon"
    />
  </div>
  <img
    class="card-img-top"
    src="https://via.placeholder.com/150"
  />
</div>
`;
